bleachbit/Common.py

```diff
@@ -139,6 +143,8 @@
     # system-wide installed locale
     if sys.platform.startswith('linux'):
         locale_dir = "/usr/share/locale/"
+    elif sys.platform.startswith('darwin'):
```

Comment:
To avoid repeating the same directory, you could also use an `or` condition after `startswith(`linux`)`